lld/test/ELF/aarch64-reloc-implicit-addend.test

```diff
@@ -11,15 +11,25 @@ REQUIRES: aarch64
 ## .reloc directives containing no addend, this succeeds.
 
 # RUN: rm -rf %t && split-file %s %t && cd %t
+
```

Comment:
Nit: Was this extra line intentional?

Reply:
Yes. The set of `RUN` lines is now divided into three: that single initial common-setup line that makes the working directory, and then two sections extremely similar to each other which do the little- and big-endian versions of the test. Without that blank line, the two similar sections would be less obviously similar, because one would contain the extra setup command.

(And I didn't want to run the setup command twice, deleting the LE version of the test dir, because if the test fails, it's useful to have _both_ sets of output files in the temp directory to examine.)

Reply:
OK, no problem. Just wanted to check.